Modules/Core/Common/test/itkBSplineKernelFunctionTest.cxx

```diff
@@ -170,17 +181,19 @@ itkBSplineKernelFunctionTest(int, char *[])
     using FunctionType = itk::BSplineKernelFunction<SplineOrder - 1>;
     auto function = FunctionType::New();
 
-    for (double xx = -3.0; xx <= 3.0; xx += 0.1)
+    for (double xx = -3.0; xx <= 3.0; xx += epsilon)
```

Comment:
epsilon is 1e-6, not 0.1

Reply:
Should have been a few lines downwards. Thanks. Fixed in 542eb7b.